src/main/java/me/jellysquid/mods/sodium/client/render/chunk/ChunkRenderManager.java

```diff
@@ -227,6 +227,10 @@ private void addEntitiesToRenderLists(ChunkRenderContainer render) {
     }
 
     public ChunkRenderContainer getRender(int x, int y, int z) {
+        if (y < 0 || y >= 16) {
```

Comment:
This should probably check the array size instead of these constants for future-proofing.
```suggestion
        if (y < 0 || y >= this.columns.length) {
```